### ISSUE TITLE
Travis CI OSS credits?

### ISSUE THREAD

**Issue by dkoslicki** [RTXteam/RTX#1413]
Looking at the [Travis CI plans](https://travis-ci.com/plans), it appears that the free plan has a limited number of credits. However, they also say credits are available for OSS projects. @finnagin Can you look into how we can get “verified” as an OSS project (eg. Find and fill the application form).

**Comment by saramsey** [comment]
Thank you for researching this, @dkoslicki

**Comment by finnagin** [comment]
So I have looked into this and the way it works is when we run low on credits we can request more via email which they grant to open source projects on a case by case basis. Currently we have used 0 credits and are currently not using any credits. I'm not sure why this is. Maybe it doesn't cost credits if we are running under a certain number of builds on public repos? Googling has not led me to any answers yet.

It's possible it only updates after a month so I'll keep watching it and see. If at some point we start using credits we can see how fast we use them and know how many to request from travis.

![plan-usage-travis-ci](https://user-images.githubusercontent.com/20198864/116128753-21515300-a67e-11eb-9ffb-f6903a18bf01.png)
